components/hr-solution/Hero.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div>
+    <!-- Above Large screen -->
+    <div class="hero-container-lg d-none d-lg-flex justify-content-around">
+      <div class="hero-left-container">
+        <nuxt-img
+          src="/hr-solution/hero-left.svg"
+          alt="Hero-Image"
+          class="img-fluid hero__image"
+        />
+      </div>
+      <div
+        class="hero-middle-container d-flex flex-column align-items-center my-auto"
+      >
+        <h1>
+          Automate your Human Resource Operations with our HTML Form Builder and
+          <span class="hglt">HR Form</span> Templates
+        </h1>
+        <a
+          href="https://app.formester.com/users/sign_up"
+          class="btn button hero__button mt-4 mb-4 mb-lg-0"
+          >Get Started for Free</a
+        >
+      </div>
+      <div class="hero-right-container">
+        <nuxt-img
+          src="/hr-solution/hero-right.svg"
+          alt="Hero-Image"
+          class="img-fluid hero__image"
+        />
+      </div>
+    </div>
+
+    <!-- Below Large screen -->
+    <div
+      class="hero-container-sm d-lg-none d-flex flex-column justify-content-center"
+    >
+      <div
+        class="hero-top-container-sm d-flex flex-column align-items-center my-auto"
+      >
+        <h1 class="hero-heading-sm">
+          Automate your Human Resource Operations with our HTML Form Builder and
+          <span class="hglt">HR Form</span> Templates
+        </h1>
+        <a
+          href="https://app.formester.com/users/sign_up"
+          class="btn button hero__button mt-4 mb-4 mb-lg-0"
+          >Get Started for Free</a
+        >
+      </div>
+      <div class="hero-bottom-container-sm d-flex justify-content-around">
+        <div class="hero-left-container-sm">
+          <nuxt-img
+            src="/hr-solution/hero-left.svg"
+            alt="Hero-Image"
+            class="img-fluid hero__image"
+          />
+        </div>
+        <div class="hero-right-container-sm">
+          <nuxt-img
+            src="/hr-solution/hero-right.svg"
+            alt="Hero-Image"
+            class="img-fluid hero__image"
+          />
+        </div>
+      </div>
```

Comment:
This can be achieved through one section itself. 
Keep the same code used for smaller devices. and for larger devices, you can use position to show them right and left respectively.